### PR TITLE
Add support for vpiMinTypMaxOp

### PR DESCRIPTION
**Pull request by kamilrakoczy**
Signed-off-by: Kamil Rakoczy <krakoczy@antmicro.com>

### CODE REVIEW

File: uhdm-plugin/UhdmAst.cc [hunk 1]
```diff
@@ -2589,6 +2589,15 @@ void UhdmAst::process_operation()
             current_node = nullptr;
             break;
         }
+        case vpiMinTypMaxOp: {
+            // ignore min and max and set only typ
+            log_assert(current_node->children.size() == 3);
+            auto tmp = current_node->children[1];
+            current_node->children.clear();
```
Comment (RRozak) [inline uhdm-plugin/UhdmAst.cc:2596]: Shouldn't we delete the nodes children[0] and children[2]?

**Review by RRozak — CHANGES_REQUESTED** [review]

File: uhdm-plugin/UhdmAst.cc (hunk 1 above)
Comment (kamilrakoczy) [reply uhdm-plugin/UhdmAst.cc]: True, now I'm cloning children[1] and using ``delete_children`` to make sure we delete all children: https://github.com/YosysHQ/yosys/blob/master/frontends/ast/ast.cc#L267

Comment (RRozak) [reply uhdm-plugin/UhdmAst.cc]: `delete children` is called by AstNode destructor: https://github.com/YosysHQ/yosys/blob/master/frontends/ast/ast.cc#L279. So it isn't needed to be called here.

Comment (kamilrakoczy) [reply uhdm-plugin/UhdmAst.cc]: Fixed.

**Review by RRozak — APPROVED** [review]

**Pull request merged**